src/dsp_tools/commands/excel2json/utils.py

```diff
@@ -73,7 +74,7 @@ def read_and_clean_all_sheets(excelfile: str) -> dict[str, pd.DataFrame]:
     try:
         return {name.strip(""): clean_data_frame(df) for name, df in df_dict.items()}
     except AttributeError:
-        msg = InvalidSheetNameProblem(excelfile, list(df_dict.keys())).execute_error_protocol()
+        msg = InvalidSheetNameProblem(str(excelfile), list(df_dict.keys())).execute_error_protocol()
```

Comment:
even better: InvalidSheetNameProblem should store the excelfile as Path, and not as str.

Reply:
If it is used as a path I understand the argument but here it is only used in the message, so I don't see the benefit.